### PR TITLE
Smaller enterprise standard image

### PR DESCRIPTION
**Pull request by jungziege**
This makes a smaller standard image, most notably by removing android.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -104,6 +104,33 @@ deploy-ubuntu-14.04-XL:
...
+	-t $(IMAGE_REPO):ubuntu-14.04-enterprise-$(VERSION) \
+	-f targets/ubuntu-14.04-enterprise/Dockerfile \
+	.
+
+push-ubuntu-14.04-enterprise:
+	docker-cache-shim push ${IMAGE_REPO}:ubuntu-14.04-enterprise-$(VERSION)
+	$(call docker-push-with-retry,$(IMAGE_REPO):ubuntu-14.04-enterprise-$(VERSION))
+
+dump-version-ubuntu-14.04-enterprise:
+	docker run $(IMAGE_REPO):ubuntu-14.04-enterprise-$(VERSION) sudo -H -i -u ubuntu /opt/circleci/bin/pkg-versions.sh | jq . > $(CIRCLE_ARTIFACTS)/versions-ubuntu-14.04-XXL-enterprise.json; true
```
Comment (kimh) [inline Makefile]: `s/versions-ubuntu-14.04-XXL-enterprise.json/versions-ubuntu-14.04-enterprise.json/`

File: Makefile [hunk 2]
```diff
@@ -104,6 +104,33 @@ deploy-ubuntu-14.04-XL:
...
+	curl -o versions.json.before https://circleci.com/docs/environments/trusty.json
+	diff -uw versions.json.before $(CIRCLE_ARTIFACTS)/versions-enterprise.json > $(CIRCLE_ARTIFACTS)/versions-ubuntu-14.04-enterprise.diff; true
+
+deploy-ubuntu-14.04-enterprise:
+	./docker-export $(IMAGE_REPO):ubuntu-14.04-enterprise-$(VERSION) > build-image-ubuntu-14.04-XXL-enterprise-$(VERSION).tar.gz
```
Comment (kimh) [inline Makefile]: `s/build-image-ubuntu-14.04-XXL-enterprise-$(VERSION).tar.gz/build-image-ubuntu-14.04-enterprise-$(VERSION).tar.gz/`

File: Makefile [hunk 3]
```diff
@@ -104,6 +104,33 @@ deploy-ubuntu-14.04-XL:
...
+	aws s3 cp ./build-image-ubuntu-14.04-enterprise-$(VERSION).tar.gz s3://circleci-enterprise-assets-us-east-1/containers/circleci-trusty-container-$(VERSION).tar.gz --acl public-read
+
+ubuntu-14.04-enterprise: build-ubuntu-14.04-XXL-enterprise push-ubuntu-14.04-XXL-enterprise dump-version-ubuntu-14.04-XXL-enterprise
```
Comment (kimh) [inline Makefile]: Ditto

**Comment by kimh** [comment]
Could you add  the following to the Makefire just to let the test to pass?

```
test-ubuntu-14.04-enterprise:
	exit 0
```

**Comment by jungziege** [comment]
I think we should still keep ubuntu-14.04-XXL-enterprise, just so that folks can easily switch to it if they want a bigger "catch-all" container.

**Comment by kimh** [comment]
LGTM

**Pull request merged**